addons/web/static/src/core/network/rpc_service.js

```diff
@@ -27,6 +27,10 @@ export class ConnectionLostError extends Error {
 
 export class ConnectionAbortedError extends Error {}
 
+export function isRequestError(e) {
+    return e instanceof RPCError || e instanceof ConnectionLostError;
```

Comment:
Can't we let the ConnectionLost handler deal with those errors? I'm wondering whether we couldn't only check for RPCErrors everywhere you use this function.

Also, currently, I think the name of the function is a bit vague.

Reply:
I tried to reproduce the `guardedCatch` check this function to have the minimal difference but yes I think we can only check for `RPCError` and let error service handle the `ConnectionLostError`